developers.cloudflare.com/wrangler.toml

```
@@ -8,7 +8,9 @@ workers_dev = false
 account_id = "b54f07a6c269ecca2fa60f1ae4920c99" # cloudflare-docs.workers.dev
 zone_id = "351cf9fc660523187714fa772ad5ca49"
 route = "https://developers.cloudflare.com/*"
-
+kv_namespaces = [
+    ( binding = "REDIRECTS" id = "42719e2c4502479e9145f347015e5b34" )
```

The syntax is slightly different:
```suggestion
    { binding = "REDIRECTS", id = "42719e2c4502479e9145f347015e5b34" }
```

---

Okay, fixed that. And as I'm addressing it, just now seeing the option to `Commit suggestion`